log/log_test.go

```diff
@@ -40,7 +41,7 @@ func TestInfoWithFields(t *testing.T) {
 		assert.Equal(t, fields["msg"], "test")
 		assert.Equal(t, fields["level"], "info")
 		assert.Equal(t, fields["key"], "value")
-		assert.Equal(t, fields["pkg"], "log")
+		assert.Equal(t, fields["pkg"], "log.TestInfoWithFields")
```

Comment:
if you change to three https://github.com/almighty/almighty-core/blob/master/log/log.go#L226 the log_test.go will show the right pkg name